tensorflow/python/ops/image_grad.py

```diff
@@ -22,6 +22,8 @@
 from tensorflow.python.framework import ops
 from tensorflow.python.ops import array_ops
 from tensorflow.python.ops import gen_image_ops
+from tensorflow.python.ops.math_ops import cast, add, reciprocal, square
```

Comment:
In tf style we don't import individual symbols, we import the whole package and use the fully qualified name, like math_ops.cast, etc.